### DISCUSSION TITLE
I cannot run the widget

### DISCUSSION THREAD

**Discussion by titaliu1224** [rudotriton/scriptable-calendar-widget#20] [Q&A]
I copied calendar.js to my scriptable and changed some variables in the readme.md.  It was worked two days ago, but there is an error code in the widget on my home screen today: 
```
Error on line 59:34: Expected value of type string but got value of type undefined.
```
I'm not sure how to fix it. Could you please tell me?

**Comment by titaliu1224** [comment]
I changed `text` to `String(text)` in line 59,  it works. Will it cause another bug?

**Reply by rudotriton** [reply]
None of the settings changes should cause this issue. It's possible that it could have been caused when creating an event's attendance status icon, so I did update the code now, so it should also always give back a string. I guess the `String(text)` would fix it, but it's possible that you get the string "undefined" displayed somewhere with that.

**Reply by titaliu1224** [reply]
Yeah, I really got an undefined icon on it! After I agreed with the invitation to that activity, it became a checked icon.
Thank you very much.
![S__1949699](https://user-images.githubusercontent.com/71343865/139597485-69ca00a6-6189-439b-b25a-1f201614b368.jpg)
